### PR TITLE
ACC-1878 Add message to non editable fields on edit form & Update “Ci…

### PR DESCRIPTION
**Pull request by sakibshaik**
Add message to non editable fields on edit form & Update “City” Field for APAC user

### Ticket
https://financialtimes.atlassian.net/browse/ACC-1878

### Screenshots

Before
screenshot-localhost_5005-2022 08 22-13_00_39
After
 | screenshot-localhost_5005-2022 08 22-12_59_41 |

### Reminder
Have you completed these common tasks (remove those that don't apply)?

- [x] **Tests** written for new or updated for existing functionality
- [x] **Stories** updated to use this change
- [x] **Design Review** ran past the designer
- [x] **Product Review** ran past the product owner

### CODE REVIEW

**Review by danieleruiz — APPROVED** [review]

**Review by Nazehs — APPROVED** [review]
looks okay to me

**Review by AniaMakes — APPROVED** [review]
I'm having a slow day. Code looks good and I can't see any obvious issues (you may want one more person to have a look) :)

File: components/delivery-city.jsx [hunk 1]
```diff
@@ -22,8 +22,14 @@ export function DeliveryCity ({
...
 	};
+	const FTShippingZone = identifyFTShippingZone(country);
 
-	const inputLabel = country === 'ARE'? 'Emirate or City':'City';
+	let inputLabel = 'City';
+	if(FTShippingZone === 'ARE'){
+		inputLabel = 'Emirate or City';
+	} else if (FTShippingZone === 'APAC'){
+		inputLabel ='City/Prefecture';
+	}
```
Comment (ludrob) [inline components/delivery-city.jsx:32]: Nothing wrong with your version (don't feel like you have to change this), just an alternative, I think closer to what is just above with the placeholder mapping, and easier to extend:

```suggestion
	const labelMapping = {
		ARE: 'Emirate or City',
		APAC: 'City/Prefecture',
		DEFAULT: 'City',
	};
	const inputLabel = labelMapping[FTShippingZone] ? labelMapping[FTShippingZone] :  labelMapping['DEFAULT]';
```

Comment (AniaMakes) [reply components/delivery-city.jsx]: Cunning! 🌟

**Review by AniaMakes — DISMISSED** [review]

**Review by danieleruiz — DISMISSED** [review]

**Review by Nazehs — APPROVED** [review]

**Review by AniaMakes — APPROVED** [review]

**Pull request merged**